src/js/themes/base.js

```diff
@@ -51,16 +51,16 @@ const focusColor = accentColors[0];
 const colors = {
   active: rgba(221, 221, 221, 0.5),
   'background-back': {
-    dark: '#33333308',
-    light: '#EDEDED',
+    light: '#33333308',
```

Comment:
I don't think background-back or background-front need to change, just background-contrast.